devcycle_python_sdk/local_client.py

```diff
@@ -103,18 +113,27 @@ def variable(self, user: DevCycleUser, key: str, default_value: Any) -> Variable
             if variable:
                 return variable
         except VariableTypeMismatchError:
-            logger.info("Variable type mismatch, returning default value")
+            logger.debug("Variable type mismatch, returning default value")
         except Exception as e:
-            logger.error("Error retrieving variable for user: %s", e)
+            logger.warning(f"Error retrieving variable for user: {e}")
```

Comment:
Do we want all of our exceptions to be the same logger level? there are some with logger.error and others with logger.warning

Reply:
yeah, i kind of reserved error for when things go bad inside the SDK and the state changed, such as if we start getting a Forbidden from the config CDN and stop the polling thread, as opposed to general network exceptions which will be retried.